src/locale/locales/de/messages.po

```diff
@@ -760,7 +760,7 @@ msgstr "Antwort verfassen"
 
 #: src/screens/Onboarding/StepModeration/ModerationOption.tsx:67
 msgid "Configure content filtering setting for category: {0}"
-msgstr "Konfiguriere die Einstellung der Inhaltsfilterung für die Kategorie: {0}"
+msgstr "Inhaltsfilterungseinstellung der Kategorie {0} konfigurieren"
```

Comment:
Because of the different sentence structure I assume there's no need for the colon here?

Reply:
yep, no need to add one